config/settings/common.py

```diff
@@ -288,7 +288,7 @@
 
 # Application Behaviors
 GITHUB_STATUS_UPDATES_ENABLED = env.bool("GITHUB_STATUS_UPDATES_ENABLED", True)
-METACI_FLOW_SUBCLASS_ENABLED = env.bool("METACI_FLOW_SUBCLASS_ENABLED", True)
+METACI_FLOW_CALLBACK_ENABLED = env.bool("METACI_FLOW_SUBCLASS_ENABLED", True)
```

Comment:
Intentionally left the env var unchanged for backwards compatibility, not sure if we're using it.

Reply:
go ahead and change the var too. the current impl is broken and off in all environments.

Reply:
Are you saying change the var here and in all the environments that turn it off, or change it here only with a new default of False?

Reply:
i'm saying 
```
METACI_FLOW_CALLBACK_ENABLED = env.bool("METACI_FLOW_CALLBACK_ENABLED", True)
```
and we can set METACI_FLOW_CALLBACK_ENABLED to False after deploying if we run into an issue.

This is meant to be a killswitch. the old impl was bad because of how it handled subflows. the flowcoordinator gets rid of that so lets plan on it working.